src/test/AludelFactoryIntegration.sol

```diff
@@ -115,269 +125,111 @@ contract AludelFactoryIntegrationTest is DSTest {
                 startTime,
                 address(crucibleFactory),
                 bonusTokens,
-                owner,
+                admin.addr(),
                 abi.encode(defaultParams)
             )
         );
 
         IAludel.AludelData memory data = aludel.getAludelData();
 
-        MockERC20(data.rewardToken).mint(owner, 1 ether);
+        Utils.fundMockToken(admin.addr(), rewardToken, REWARD_AMOUNT);
 
-        cheats.startPrank(owner);
-        MockERC20(data.rewardToken).approve(address(aludel), 1 ether);
-        aludel.fund(1 ether, 1 days);
-        aludel.registerVaultFactory(address(template));
-        cheats.stopPrank();
+        // todo : add fund aludel helper function
```

Comment:
this wasn't implemented. Since it's a small thing I'd advise to do it here, but feel free to create an issue to tackle it later

Reply:
Implemented.